### PR TITLE
fix: kr shipengine fixes

### PR DESCRIPTION
**Pull request by hrwX**
- Typecast float to int.
- `saturday_delivery` should be either True/False and not None

### CODE REVIEW

File: shipment_management/shipengine/api.py [hunk 1]
```diff
@@ -295,7 +295,7 @@ def get_shipping_rates(from_address, to_address, package, doc, items, confirmati
 			},
 			"insurance_provider": "third_party",
 			"advanced_options": {
-				"saturday_delivery": doc.get("saturday_delivery", False) if doc else False
+				"saturday_delivery": doc.get("saturday_delivery", False) if doc and doc.get("saturday_delivery") else False
```
Comment (Alchez) [inline shipment_management/shipengine/api.py]: Just a drive-by suggestion :sweat_smile: 

```suggestion
				"saturday_delivery": doc and doc.get("saturday_delivery") or False
```

**Review by sahil28297 — APPROVED** [review]
